R/operators.R

```diff
@@ -1079,22 +1079,23 @@ path <- function(...) {
   } else if ("igraph.vertex" %in% class(e2)) {
     ## Adding vertices, possibly with attributes
     ## If there is a single unnamed argument, that contains the vertex names
-    wn <- which(names(e2) == "")
-    if (length(wn) == 1) {
-      names(e2)[wn] <- "name"
+    unnamed_elements_indices <- which(!rlang::have_name(e2))
```

Comment:
I guess wn stands for which not named or something like that.

Reply:
and I find `!rlang::have_name()` more readable than `names(x) == ""`. https://masalmon.eu/2023/11/06/functions-dealing-with-names/

Reply:
They are different, though, if the vector is unnamed.

```suggestion
    unnamed_elements_indices <- which(names(e2) == "")
```